apps/sms/index.html

```diff
@@ -71,18 +71,18 @@
     <script defer src="shared/js/fb/fb_data_reader.js"></script>
     <script defer src="shared/js/fb/fb_reader_utils.js"></script>
     <script defer src="shared/js/event_dispatcher.js"></script>
-    <script defer src="js/navigation.js"></script>
-    <script defer src="js/inter_instance_event_dispatcher.js"></script>
-    <script defer src="js/time_headers.js"></script>
-    <script defer src="js/contacts.js"></script>
-    <script defer src="js/drafts.js"></script>
-    <script defer src="js/threads.js"></script>
-    <script defer src="js/utils.js"></script>
+    <script defer src="views/shared/js/navigation.js"></script>
```

Comment:
eventually navigation will really be split in 2 parts, but well, it can be wherever :)